build/charts/theia/provisioning/datasources/create_table.sh

```diff
@@ -69,10 +69,10 @@ clickhouse client -n -h 127.0.0.1 <<-EOSQL
         trusted UInt8 DEFAULT 0
     ) engine=MergeTree
     ORDER BY (timeInserted, flowEndSeconds)
-    TTL timeInserted + INTERVAL 1 HOUR
-    SETTINGS merge_with_ttl_timeout = 3600;
+    TTL timeInserted + INTERVAL {{ .Values.clickhouse.ttl }} SECOND
+    SETTINGS merge_with_ttl_timeout = {{ .Values.clickhouse.ttl }};
 
-    CREATE MATERIALIZED VIEW flows_pod_view
+    CREATE MATERIALIZED VIEW IF NOT EXISTS flows_pod_view
```

Comment:
This also seems a change not related to helm charts.

Reply:
This is related to the ClickHouse PV support. PV support is also added into this PR as

- There are only a few customization variables before we add PV support
- PV support contents have been reviewed at Antrea side. Adding PV to this PR mainly requires reviews on the Helm related contents.